config/locales/oc.yml

```diff
@@ -517,10 +539,13 @@ oc:
     settings: Paramètres
     two_factor_authentication: Autentificacion en dos temps
     your_apps: Vòstras aplicacions
+themes:
```

Comment:
add indent

Reply:
Done, it worked! Thank you for pointing where I had to make changes!